cmd/snap/cmd_quota.go

```diff
@@ -104,11 +109,13 @@ func init() {
 	cmd := addCommand("set-quota", shortSetQuotaHelp, longSetQuotaHelp,
 		func() flags.Commander { return &cmdSetQuota{} },
 		waitDescs.also(map[string]string{
-			"memory":  i18n.G("Memory quota"),
-			"cpu":     i18n.G("CPU quota"),
-			"cpu-set": i18n.G("CPU set quota"),
-			"threads": i18n.G("Threads quota"),
-			"parent":  i18n.G("Parent quota group"),
+			"memory":             i18n.G("Memory quota"),
+			"cpu":                i18n.G("CPU quota"),
+			"cpu-set":            i18n.G("CPU set quota"),
+			"threads":            i18n.G("Threads quota"),
+			"journal-size":       i18n.G("Journal size quota"),
+			"journal-rate-limit": i18n.G("Journal rate limit quota"),
```

Comment:
this should describe the expected format of the option?

Reply:
I probably need some help here, I have a hard time figuring out where these are shown? Would `Journal rate limit as <message count>/<message period>` be fine or how do we expect those formatted?

Reply:
yes, that would work

Reply:
Thanks!